Library/Formula/racer.rb

```diff
@@ -0,0 +1,45 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+  url "https://github.com/phildawes/racer/archive/v1.0.0.tar.gz"
+  sha256 "78895296ed688eeccbaf7745235f0fc503407bfa718f53583a4dcc9e1246b7f5"
+
+  keg_only "You should set path in your special editor which use racer"
+
+  option "without-src", "Do not get rust source for racer from www.rust-lang.org"
```

Comment:
Racer need the rust source code to work properly. It will search the rust source code and generate the code completion. But `rust` formula don't offer the rust source code. Users must download rust source code to use `racer`. 
So, it is necessary to add rust source code here.